apps/rod/fields/index.js

```diff
@@ -1,6 +1,5 @@
 const dateComponent = require('hof').components.date;
 const countries = require('hof').utils.countries();
-
 const validators = require('hof/controller/validation/validators');
 
 // TODO: Move this into behavior when the custom validation file is added as part
```

Comment:
I think these validations can be moved to custom-validation.js behaviour

Reply:
Hi Vinodha. Yes, I'm waiting for another [PR](https://github.com/UKHomeOffice/return-of-documents/pull/49) to be merged and after that Im planning put these changes in the custom validation. Just to avoid the conflicts